auth-web/src/models/business.ts

```diff
@@ -11,6 +11,12 @@ export interface FolioNumberload {
     businessIdentifier: string
     folioNumber: string
 }
+
+export interface CorpType {
+    code: string
+    desc: string
```

Comment:
I'm not totally familiar with this repo, but this prop name doesn't give me a lot of info on the significance of it with regards to a `CorpType`. What does it do exactly?

Reply:
I don't know. I simply moved this interface block up in the code.
Declaration order doesn't matter, but I prefer to keep related declarations together :)

Reply:
fair enough :)